docs/running-jobs/slurm-commands.md

```diff
@@ -7,10 +7,10 @@ well as provide details on how to use them.
 ## Finding queuing information with `squeue`
```

Comment:
On this page there is this section and I hate the dots, I wonder if it will be better to make this into tabs? 

![Screenshot 2024-09-19 at 4 15 10 PM](https://github.com/user-attachments/assets/adfcb438-bbd2-41b8-ad40-97b7941712cd)

Reply:
In this case, the content is too sparse to place into separate tabs. This would also hide potentially useful information in separate tabs, which users often do not switch between. I would prefer to not set these as tabs. We can consider a different format though that removes the ellipses. I rewrote the section a little to instead to use bolded identifiers for each code snippet.